### PR TITLE
Add the installation and update steps in the README #63

### PR DESCRIPTION
**Pull request by BishoyHanyRaafat**
#63

### CODE REVIEW

**Comment by BishoyHanyRaafat** [comment]
@mark-at-pieces merge

**Review by shivay-at-pieces — APPROVED** [review]
looks good

File: README.md [hunk 1]
```diff
@@ -280,7 +282,24 @@ Note: Ensure you get latest from the [releases](https://github.com/pieces-app/cl
...
-these can be local/github/pypi etc.
\ No newline at end of file
+these can be local/github/pypi etc.
+
+
+### Updating
+To update the project, run the following command:
+
+```shell
+pip install pieces --upgrade
```
[the remark this answers is not in the document]
Comment (BishoyHanyRaafat) [reply README.md:295]: Yeah so u should write the name of the whl instead

**Comment by BishoyHanyRaafat** [comment]
Should I remove the update part?

**Review by hal-8999-alpha — COMMENTED** [review]
Are these placeholders for later commands or are the commands already implemented?

**Pull request merged**